test/quantization/test_quantized_op.py

```diff
@@ -456,6 +456,43 @@ def test_qclamp(self, X, min_val, max_val):
             qY_hat = op(qX, min_val, max_val)
             self.assertEqual(qY, qY_hat, message="{} qclamp failed".format(name))
 
+    """Tests the correctness of the quantized::clamp_with_tensors op."""
+    @given(X=hu.tensor(shapes=hu.array_shapes(1, 8, 1, 8),
+                       elements=hu.floats(-1e6, 1e6, allow_nan=False),
+                       qparams=hu.qparams()))
+    def test_qclamp_with_tensors(self, X):
+        X, (scale, zero_point, torch_type) = X
+        min_tensor = np.random.uniform(-1e5, 0, size=X.shape).astype(np.float32)
+        max_tensor = np.random.uniform(0, 1e5, size=X.shape).astype(np.float32)
+
+        Y = X.copy()
```

Comment:
optional: we can also reference the floating point implementation here, instead of reimplementing the calculation.  It doesn't make much difference at the current state, but might make things easier once we add handling for different quantization params.

Reply:
Now reference the floating point implementation in the tests.